### ISSUE TITLE
question about  Alignment Restricted Transducer 

### ISSUE THREAD

**Issue by jinggaizi** [pytorch/audio#2011]
i saw beta version of transducer loss relies on code from Alignment Restricted Streaming Recurrent Neural Network Transducer , maybe it's has parameters the left and right buffers bl/lr, how can i use the feature of this loss?

**Comment by carolineechen** [comment]
Hi @jinggaizi, the transducer loss in torchaudio was adapted from the code from the AR-RNNT paper, but the torchaudio version currently only supports a vanilla version, without the additional proposed features from the paper. We may consider adding support for these additional features in the future.

**Comment by jinggaizi** [comment]
thanks for your quick reply
